.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+## What kind of a pull request is this?
+
+- [ ] QA / Code Review
+<!-- Add other options if appropriate -->
+
+
+## Code Checklist
+
+- [ ] The code follows dotKom code style 
```

Comment:
On a serious note though, we should probably rename the organization to conform with Online's statutes (github redirects you to the old name anyways, unless someone re-registers it -- but i don't think people can register a new organization with the same name using different casing.)